extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -975,7 +976,7 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     const iframeDoc = getFrameDoc(iframe);
     const tags = getA4AMetaTags(iframeDoc);
     const vars = {};
-    tags.forEach(tag => {
+    iterateCursor(tags, tag => {
```

Comment:
There's a conflict between this line and #23559. Your PR won the race by a few seconds, so I've sent out a fix in #23584